Misc build updates

mhutch opened the pull request.

slluis commented:
The build has several issues: https://wrench.internalx.com/Wrench/ViewLane.aspx?lane=monodevelop-lion-monodevelop-pr-1589&host_id=243&revision_id=790232

NuGet mismatches and unexpected duplicates of .config files (the Workspaces.dll duplicates are also an issue in master).

mhutch commented:
I can't fix those in a PR against this repo though

mhutch commented:
Updated duplicates checks in https://github.com/xamarin/md-addins/pull/1164

Removed the Newtonsoft.Json update since it's complex to sync across all repos. WIll live with the current one for now.
